tools/hardware/utils/memory.py

```diff
@@ -61,11 +61,6 @@ def align_memory(regions: Set[Region], config: Config) -> List[Region]:
     ret = sorted(regions)
     extra_reserved = set()
 
-    if config.arch == 'riscv':
-        # RISC-V is special: it expects physBase to be
-        # the address that the bootloader is loaded at.
-        physBase = ret[0].base
```

Comment:
How come this doesn't need to be set before the following two if statements anymore? It looks like it still needs separate treatment compared to Arm.

Reply:
Not sure this was from a month or so ago. It does look like those ifs are needed, will remove.

Reply:
I've refactored align_memory into the config class. I think this makes it much clearer as to what is going on.

Reply:
According to @kent-mcleod, this ist kind of RISC-V hack that https://github.com/seL4/seL4/pull/365 will resolve then.